Marlin/Regulator.cpp

```diff
@@ -120,12 +131,19 @@ static void pressure_protection(float target_pressure, float pressure) {
       _regulator_error_handler(PSTR(MSG_T_PNEUMATICS_EREG_ABOVE_PUMP), pressure);
     }
   
-    // Calculate the conditions where we may enter an error.
-    bool regulatorUnder =
-        (pressure <= (target_pressure - REGULATOR_PROTECTION_BAND));
-    bool regulatorOver =
-        (pressure >= (target_pressure + REGULATOR_PROTECTION_BAND));
-  
+    // Calculate the conditions where we may enter an error. This is split into
+    // two bands, to accomodate larger variable in higher pressures.
+    if (pressure <= REGULATOR_BAND_CROSSOVER_PSI) {
```

Comment:
"pressure" should probably be "target_pressure" to make this more consistent in operation.